srv/GetInformJob.srv

```diff
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: 2024, Yaskawa America, Inc.
+# SPDX-FileCopyrightText: 2024, Delft University of Technology
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# NOTE:
+# - do not include the file extension (JBI), only the name of the job
+# - character encodings other than ASCII are only partially supported
+# - a job with this name must exist on the controller
+# - a job with this name must not be currently running on the controller
+string name
+
+---
+
+# Result of the service invocation. Values other than one (1) signal failure.
+#
+# NOTE: future versions of this service may use a different set of result codes
+uint32 result_code
+
+# string representation of the value in 'result_code', for humans
+string message
+
+# this field may be left uninitialised in case result_code != 1
+byte[] contents
```

Comment:
I made this a `byte` field, as it seemed more robust/future proof than a `string` (we're CRUD-ing files, not strings).

Unfortunately, `byte[]` becomes `list[bytes]` in Python (that's a list of `bytes` objects, which themselves have only a single byte each), which makes it really uncomfortable to work with.

I'll see if we should switch to `uint8[]`.

Reply:
I've switched to `uint8` and so far it's been more convenient than `byte`.